terraform/cluster/aws/docker-swarm/dev/variables.tf

```diff
@@ -63,15 +63,40 @@ variable "worker_zone" {
 
 variable "total_manager_instances" {
   description = "Number of Managers"
-  default     = "3"
```

Comment:
Keep in mind that if we reduce managers we are risking the cluster (so better to leave those to 3). We can use the managers as workers as well